src/main/features/claimant-response/routes/payment-plan.ts

```diff
@@ -53,13 +53,23 @@ class PaymentPlanPage extends AbstractPaymentPlanPage<DraftClaimantResponse> {
     const claim: Claim = res.locals.claim
     const draft: Draft<DraftClaimantResponse> = res.locals.draft
     const courtDecision = CourtDecisionHelper.createCourtDecision(claim, draft)
+    const claimResponse = claim.response as FullAdmissionResponse | PartialAdmissionResponse
```

Comment:
`const admissionResponse: FullAdmissionResponse | PartialAdmissionResponse = ...`

Reply:
Done. Added type for claimResponse and courtDecision in previous line